Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -4,151 +4,314 @@
 
```

Comment:
```suggestion
from typing import *
```

Reply:
It is used to declare a type in the signature of the functions, for example: `Dict[str, Any]` in line 288
`Dict` is in the `typing` library